src/blocks/file-blocks/explain/explanation.tsx

```diff
@@ -23,23 +24,25 @@ export function ExplanationComponent(props: {
 
   return (
     <>
-      <div className="sticky top-0 z-10 pb-2 bg-white">
-        <p className="text-xs font-mono">
+      <div className={tw(`sticky top-0 z-10 pb-2 bg-white`)}>
+        <p className={tw(`text-xs font-mono`)}>
           Explanation for{" "}
-          <span className="font-bold">
+          <span className={tw(`font-bold`)}>
             L{explanation.start}:L
             {explanation.end}
           </span>
         </p>
       </div>
 
-      <div className="relative">
+      <div className={tw(`relative`)}>
         {status === "loading" && (
-          <p className="text-xs  font-mono animate-pulse">Loading...</p>
+          <p className={tw(`text-xs  font-mono animate-pulse`)}>Loading...</p>
         )}
         {status === "success" && data && (
-          <div className="pb-2">
-            <p className="text-xs whitespace-pre-line font-mono">{data}</p>
+          <div className={tw(`pb-2`)}>
```

Comment:
this one might be a Primer style?

Reply:
How do you mean? Aren't these Tailwind utility classes too?

Reply:
yep! I'm guessing at what scale the author (probably me in this case) was aiming for - if Primer has been overriding the Tailwind classes, switching to the `pb-2` Tailwind class would be too small. I could definitely be missing somethign!

Reply:
Totally defer to you in this case